app/controllers/comments_controller.rb

```diff
@@ -7,6 +7,8 @@ def new
 
   def create
     @comment = Comment.new(comment_params)
+    @comment.user_id = current_user.id  # Set the user_id explicitly
+
```

Comment:
![image](https://github.com/AlyAtalla/blog-app/assets/93039359/0a40d69e-8e1b-40cf-a9fd-11f073dd0db9)

- [ ] It would be nice if you fix the problem coming from post creation which the previous reviewer already mentioned 🤔 Kindly fix it to make your app work as expected 😊 If you need any help then you can reach out to me on slack without any hesitation 👍